src/lexer/token.h

```diff
@@ -17,6 +17,73 @@ typedef enum {
     TT_CBRACKET, // ]
     TT_SEMI, // ;
     TT_NO_TOKEN, // Not a token
+	
```

Comment:
I thought of a couple more that need to be added:
```c
TT_XOR, // ^
TT_XOREQ // ^=
TT_POINT, // ->
```

Reply:
Just realized -- also `TT_LEFTSHIFTEQUALS` and `TT_RIGHTSHIFTEQUALS` for `<<=` and `>>=`.

Reply:
Oh one more!!! (Hopefully this is the last one) there's also the dot operator `.`.